docs/framework/react/guide/server-functions.md

```diff
@@ -0,0 +1,245 @@
+---
+id: server-functions
+title: Server Functions
+---
+
+As [TanStack Start](./tanstack-start.md) is a full-stack framework, we need a way to write code that executes on the
```

Comment:
I am not sure if it works with the rest of the document flow, but would likening it to dynamic-RPC could help readers understand server functions better.

Purely adding this in here, because of the mental model that people need to keep in mind, since these aren't just API-endpoints per-se. Rather, little portals through which server-side code can be executed through an API-surface the user controls.

Since currently our server functions are also "server actions", I just feel like we either need to explain a bit more from the get-go or possibly link to external resources.

Reply:
> I just feel like we either need to explain a bit more from the get-go

Let's do that - explain a bit more from the get-go.

Could you possibly work on adding in a section that explains what the differences are and the mental model folks need to be in when using these?

Reply:
@crutchcorn I think what I've added gets the message across. Feel free to cut out any bits that feel a bit repetitive.